packages/components/pie-chip/test/component/pie-chip.spec.ts

```diff
@@ -79,4 +83,171 @@ test.describe('PieChip - Component tests', () => {
+                });
+            });
+
+            test.describe('when an aria close value is not provided', () => {
+                test('should not render on the component', async ({ mount, page }) => {
+                    // Arrange
+                    await mount(PieChip, {
+                        props: {
+                            ...dismissibleProps,
+                            aria: {
+                                close: '',
+                            },
+                        } as ChipProps,
+                    });
+
+                    const chip = page.locator(componentSelector);
+                    const closeButton = chip.locator(closeBtnSelector);
+
+                    // Act && Assert
+                    await expect(closeButton).not.toHaveAttribute('aria-label', 'Chip Close');
+                });
+            });
+        });
+    });
+
+    test.describe('Chip: aria: attributes', () => {
+        test.describe('aria-live', () => {
+            test('should render on the component with the value `polite`', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieChip, {
+                    props,
+                });
+
+                const chip = page.locator(componentSelector);
+
+                // Act && Assert
+                await expect(chip).toHaveAttribute('aria-live', 'polite');
+            });
+        });
+
+        test.describe('aria-atomic', () => {
+            test('should render on the component with the value `true`', async ({ mount, page }) => {
+                // Arrange
+                await mount(PieChip, {
+                    props,
+                });
+
+                const chip = page.locator(componentSelector);
+
+                // Act && Assert
+                await expect(chip).toHaveAttribute('aria-atomic', 'true');
+            });
+        });
+
+        test.describe('aria-label', () => {
+            test.describe('when passed in', () => {
+                test('should render on the component with the correct value', async ({ mount, page }) => {
+                    // Arrange
+                    await mount(PieChip, {
+                        props,
+                    });
+
+                    const chip = page.locator(componentSelector);
+
+                    // Act && Assert
+                    await expect(chip).toHaveAttribute('aria-label', 'Chip Label');
+                });
+            });
+
+            test.describe('when not passed in', () => {
+                test('should not render on the component', async ({ mount, page }) => {
+                    // Arrange
+                    await mount(PieChip, {
+                        props: {} as ChipProps,
+                    });
+
+                    const chip = page.locator(componentSelector);
+
+                    // Act && Assert
+                    await expect(chip).not.toHaveAttribute('aria-label', 'Chip Label');
```

Comment:
It looks like this is checking only for this specific value, so if there was another value the test would still pass?

This might not be exact code, but I think I would prefer something like this:
```js
await expect(chip.attributes()).not.toContain('aria-label');
```